pyproject.toml

```diff
@@ -38,6 +38,10 @@
 
 
 [dependency-groups]
+dev = [
+  "pre-commit>=4.0.1",
+  { include-group = "test" },
```

Comment:
spacing style does not match with `project.authors`

Reply:
Done.